error/src/lib.rs

```diff
@@ -581,11 +581,13 @@ impl CaliptraError {
     pub const KAT_HMAC384_FAILURE: CaliptraError = CaliptraError::new_const(0x90030001);
     pub const KAT_HMAC384_TAG_MISMATCH: CaliptraError = CaliptraError::new_const(0x90030002);
 
-    pub const KAT_ECC384_SIGNATURE_GENERATE_FAILURE: CaliptraError =
-        CaliptraError::new_const(0x90040001);
-    pub const KAT_ECC384_SIGNATURE_VERIFY_FAILURE: CaliptraError =
-        CaliptraError::new_const(0x90040002);
+    // 0x90040001 was KAT_ECC384_SIGNATURE_GENERATE_FAILURE
+    // 0x90040002 was KAT_ECC384_SIGNATURE_VERIFY_FAILURE
```

Comment:
Wondering if we should also mark this as deprecated in error/README.md.

Reply:
That's a good question. It doesn't look like that file has really been maintained unfortunately. It's very out-of-sync with the lib.rs source. We should probably either remove that or auto-generate it.

Reply:
This table has been removed now in https://github.com/chipsalliance/caliptra-sw/pull/1779 after discussion in the FW sync today